### DISCUSSION TITLE
Workflow Description limit

### DISCUSSION THREAD

**Discussion by jbbrown333** [nathanwoulfe/Plumber-2#126] [Q&A] [answered]
I noticed recently that there is a 255 char limit on the workflow description field. When entering more than 255 char, it doesn't throw an error and it doesn't activate the workflow. Is there a way to either increase that limit - or - throw an error so the user knows they've surpassed that limit - or - just have a note above that text area to let users know that there is a limit and for longer descriptions, they should include an attachment? Or are there other ideas that you may have? Thank you!

**Comment by nathanwoulfe** [comment, marked as answer]
Hey @jbbrown333 that sounds like a bug - the logic all exists for showing a character count on the comment field, but it sounds like it's not displaying. 

Shouldn't be a big deal to fix, I'll aim to get it into a patch for 8/9 alongside the Umbraco 10 package next week. 

Thanks for letting me know 👍

**Reply by jbbrown333** [reply]
Thank you so much! I'll keep my eye out for the next release.

**Reply by nathanwoulfe** [reply]
2.1.6 is on the way to Nuget now 😄

**Reply by nathanwoulfe** [reply]
I was partially correct in that the logic exists, but the initiate workflow view wasn't using the comments component used elsewhere. Updated to use the component, improved some logic, all should be peachy.
